package.json

```diff
@@ -9,23 +9,24 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "chart.js": "3.3.2",
+    "file-saver": "^2.0.5",
+    "final-form": "^4.20.2",
+    "formik": "^2.2.6",
+    "isomorphic-dompurify": "^0.17.0",
+    "jspdf": "^2.4.0",
+    "jspdf-autotable": "^3.2.5",
     "next": "12.0.4",
+    "primeflex": "2.0.0",
+    "primeicons": "^5.0.0",
+    "quill": "1.3.7",
     "react": "17.0.2",
```

Comment:
When I did NPM INSTALL of isomorphic-dompurify it sorted the deps alphabetically.